recipes/obspy/meta.yaml

```diff
@@ -0,0 +1,69 @@
+{% set version = "1.0.2" %}
+{% set sha256 = "37d563e36a44a5309b2f53b394d7391d49c8a62b0e8d0f0d13f54ea4e2943230" %}
+# {% set md5 = "b1432aa11b6d244c4fac61737c0c9d95" %}
```

Comment:
Fair enough. If you have a future use for them leave them here. I thought they were left-overs from the tests.